src/mpid/ch4/src/ch4_progress.c

```diff
@@ -5,6 +5,8 @@
 
 #include "mpidimpl.h"
 
+#define MPIDI_CH4_GLOBAL_POLL_FREQ 100
```

Comment:
We'd want this to be a CVAR since the optimal frequency of global progress is dependent on the application.

Reply:
Yes.